LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -618,9 +620,20 @@ - (void)_dismissMe {
 #pragma mark - UI setup
 - (void)_setupNavBarWithLogoutTitle:(NSString *)logoutTitle {
     // Navigation Bar with custom UI
-    self.navBar =
-    [[UINavigationBar alloc] initWithFrame:CGRectMake(0, LTHMainWindow.frame.origin.y,
-                                                      LTHMainWindow.frame.size.width, 64)];
+    if (LTHiOS11) {
+        UIView *patchView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, LTHMainWindow.frame.size.width, [LTHPasscodeViewController getStatusBarHeight])];
```

Comment:
Both the patch and the `navBar` could use an `autoresizingMask` of `UIViewAutoresizingFlexibleWidth`, otherwise they are both too short when moving to landscape. I know the `navBar` was already missing it, so we're also fixing an existing bug while we're at it. Could you add it for patch and also for the `navBar`, right below their respective initializations, please?